### PR TITLE
Use production HTM in production

### PR DESCRIPTION
**Pull request by developit**

### CODE REVIEW

**Review by JoviDeCroock — APPROVED** [review]

**Comment by developit** [comment]
Something's likely up here, I would have expected a non-zero filesize reduction for the prod bundles.

**Review by marvinhagemeister — APPROVED** [review]

File: src/bundler.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ export async function bundleProd({
 			htmlEntriesPlugin({ cwd, publicDir, publicPath: '/' }),
 			publicPathPlugin({ publicPath: '/' }),
 			aliasesPlugin({ aliases, cwd: root }),
-			htmPlugin(),
+			htmPlugin({ production: true }),
```
Comment (cristianbote) [inline src/bundler.js:101]: By default this is set to true inside here https://github.com/preactjs/wmr/blob/3fbed71240cd1ae252c2af68192b9b0fb0cad4aa/src/plugins/htm-plugin.js#L13. so you

Comment (developit) [reply src/bundler.js:101]: (heh, yes you are right)

**Comment by developit** [comment]
Fix is at developit/htm#189

**Comment by cristianbote** [comment]
@developit pushed a test as well to keep track of this in the future. 👍

**Comment by developit** [comment]
Aces! Thanks for the test @cristianbote.

**Pull request merged**